app/apollo/apollo-octopus-public/src/main/graphql/com/hedvig/android/apollo/octopus/graphql/termination/QueryContractsToTerminate.graphql

```diff
@@ -0,0 +1,26 @@
+query ContractsToTerminate {
```

Comment:
Since this is a new one, as I understand this is only used in that one screen.
Could we move this query inside that module instead of having it accessible by all feature modules in the app?
The naming would also not make sense querying it from inside :feature-home for example.
Putting it in there should fix this issue.